modules/end-user-guide/pages/url-parameter-for-workspace-image.adoc

```diff
@@ -0,0 +1,14 @@
+:_content-type: CONCEPT
+:description: URL parameter for the workspace image
+:keywords: parameter, URL, workspace, image
+:navtitle: URL parameter for the workspace image
+//:page-aliases: url-parameter-for-workspace-image.adoc
+
+[id="url-parameter-for-workspace-image"]
+= URL parameter for the workspace image
+
+You can use the `image` parameter to add a path to a container image in the following scenarios:
```

Comment:
I'm just thinking how I could use this parameter if I were a user.
Do I need to use it with an equals sign or a colon?
I suggest asking the developer to provide a couple of examples that you can use to abstract the image URL from and rewrite it as a _<user-filled_value>_ and then create a code block for clarity. For example (I'd clear this with the developer first):

```suggestion
When you start a workspace by using a URL, you can specify additional containers to run with it by supplying the paths to those container images as part of the URL.

The URL parameter for a path to a container image is `image=`:

[subs="+quotes"]
----
image=https://__<url_to_the_image>__
----

You can use this parameter in the following scenarios:
```